specification/applicationinsights/data-plane/LiveMetrics/preview/2024-04-01-preview/livemetrics.json

```diff
@@ -0,0 +1,1213 @@
+                "type": "string",
+                "description": "A boolean flag indicating whether there are active user sessions 'watching' the instrumentation key. If true, the client must start collecting data and posting it to Live Metrics. Otherwise, the client must keep pinging."
+              },
+              "x-ms-qps-configuration-etag": {
+                "type": "string",
+                "description": "An encoded string that indicates whether the collection configuration is changed."
+              },
+              "x-ms-qps-service-polling-interval-hint": {
+                "type": "string",
+                "description": "Recommended time (in milliseconds) before the client should ping the service again. This header exists only when ikey is not subscribed to."
+              },
+              "x-ms-qps-service-endpoint-redirect-v2": {
+                "type": "string",
+                "description": "Contains a URI of the service endpoint that the client must permanently use for the particular resource. This header exists only when the client is talking to Live Metrics global endpoint."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/ServiceError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Live Metrics client checks if its data is subscribed to.": {
+            "$ref": "./examples/LiveMetrics_IsSubscribed.json"
+          }
+        }
+      }
+    },
+    "/QuickPulseService.svc/post": {
+      "post": {
+        "operationId": "Publish",
+        "description": "Publish live metrics to the Live Metrics service when there is an active subscription to the metrics.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.ikey"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.apiKey"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.configurationEtag"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.transmissionTime"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.monitoringDataPoints"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/CollectionConfigurationInfo"
+            },
+            "headers": {
+              "x-ms-qps-subscribed": {
+                "type": "string",
+                "description": "Tells the client whether the input ikey is subscribed to."
+              },
+              "x-ms-qps-configuration-etag": {
```

Comment:
If this is a standard HTTP ETag then it should use the header name "etag" -- and if it isn't, please don't include "etag" in the header name because that will be confusing.